### PR TITLE
Fix JS caching issue

### PR DESCRIPTION
**Pull request by imtiazAhmd**
#### What
Improve web caching of Front End assets
#### Ticket

[Fix application assets caching issue](https://dsdmoj.atlassian.net/browse/CFP-393)

#### Why
Users are affected by stale caches upon production releases that contain Front End asset changes and cause issues where user journeys are affected.

#### Acceptance criteria
**Given** we release Front End asset changes
**When** users make a web request
**Then** they should immediately get the latest Front End assets

--------

### CODE REVIEW

File: spec/support/jasmine-browser.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "srcDir": ".",
   "srcFiles": [
-    "public/packs/js/application.bundle.js",
-    "public/packs-test/js/application.bundle.js"
+    "public/packs-test/js/application-bundle-*.js"
```
Comment (jsugarman) [inline spec/support/jasmine-browser.json:4]: i thought you were looking to get rid of `packs-test`

Comment (naseberry) [reply spec/support/jasmine-browser.json:4]: We are keeping it for now 👍

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -174,4 +174,15 @@ def tab_matches(path)
     tab = tab_present(path)
     request.GET[:tab] == tab
   end
+
+  def app_javascript_include_tag(path = nil)
```
Comment (jrmhaig) [inline app/helpers/application_helper.rb]: `path` should be a required parameter as it is required for `Dir.glob`.

```suggestion
  def app_javascript_include_tag(path)
```

**Review by jsugarman — APPROVED** [review]
👍 Looks good and works fine on dev-lgfs

Minor tweaks possible but can do after the fact

```
    # sort array by last modified
    file = files.sort { |a, b| File.mtime(b) <=> File.mtime(a) }.first

    # return the last modified file
    javascript_include_tag file.gsub(/^public/, ''), defer: true
```

**Pull request merged**